1.9/overview/architecture/boot-sequence.md

```diff
@@ -0,0 +1,44 @@
+---
+post_title: Boot Sequence
+nav_title: Boot Sequence
+menu_order: 6
+---
+
+During DC/OS installation, the components come online in a particular sequence due to interdependencies.
+
+## Master nodes
+
+On each master node the following happens, in chronological order:
+
+1. [Exhibitor](https://github.com/mesosphere/exhibitor-dcos) starts up, creates ZooKeeper configuration and launches ZooKeeper.
+1. The Mesos masters are launched, register with the local ZooKeeper (127.0.0.1), and discover the other masters.
+1. Mesos-DNS is launched on every master node.
+1. Mesos-DNS keeps hitting `leader.mesos:5050/master/state.json`. The DNS entry `leader.mesos` points to the currently leading Mesos master.
```

Comment:
Should we mention that Mesos-DNS hits `leader.mesos:5050/master/state.json` in line 15, since we mention that it continues to do so here?